src/AggregateFunctions/AggregateFunctionTopK.cpp

```diff
@@ -7,18 +7,20 @@
 #include <DataTypes/DataTypeDateTime.h>
 
 
-#define TOP_K_MAX_SIZE 0xFFFFFF
+constexpr UInt64 TOP_K_MAX_SIZE = 0xFFFFFF;
```

Comment:
Then it should be static. Otherwise the symbol is exported and pollutes the symbol table.